manifests/tekton/tasks/base/scos-build.yaml

```diff
@@ -21,9 +32,10 @@ spec:
         devices.kubevirt.io/kvm: "1"
     script: |
       #!/usr/bin/env bash
-      #
-      echo $(params.version) 
       set -euxo pipefail
+
```

Comment:
add this in the script as its used in the OKD COPR setup

`basearch=x86_64`

Reply:
I've escaped the `$` - this is auto-substituted by the RPM stack